hashintel/hash
# Create type modal shouldn't show when clicking insert type immediately after creating a type

## 🌟 What is the purpose of this PR?




Currently, if you click "insert property/link" after you just inserted one by creating it, you'll see the create modal instead of the type selector. This PR fixes that.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203623179643287/1204062318311861/f) _(internal)_

## 🔍 What does this change?




Close modal when the insert row is hidden

## ❓ How to test this?



1. Click insert new property/link
2. Create property/link
3. Click insert again
4. Ensure see type selector and not modal
